src/site/includes/header.html

```diff
@@ -59,7 +55,10 @@
 <script defer src="/generated/static-pages.entry.js"></script>
 {% endif %}
 
-<!-- We participate in the US government’s analytics program. See the data at analytics.usa.gov. -->
+<!--
+  We participate in the US government’s analytics program. See the data at analytics.usa.gov.
+  https://github.com/digital-analytics-program/gov-wide-code
+-->
```

Comment:
Added a link to the GH repo for the DAP code. I feel like this script link was pretty mysterious to everyone.